src/form/Form.jsx

```diff
@@ -75,6 +75,8 @@ function Form({
   onSkip = null,
   onError = null,
   onCustomAction = null,
+  onView = null,
+  onViewFields = [],
```

Comment:
maybe onViewElements instead? visibility callback could apply to non-field elements too